scripts/generate_tarball.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/bash
+
+NAME=$1
+EOS_PREFIX=${PREFIX}/${SUBPREFIX}
+mkdir -p ${PREFIX}/bin/
+#mkdir -p ${PREFIX}/lib/cmake/${PROJECT}
+mkdir -p ${EOS_PREFIX}/bin 
+#mkdir -p ${EOS_PREFIX}/include
+#mkdir -p ${EOS_PREFIX}/lib/cmake/${PROJECT}
+#mkdir -p ${EOS_PREFIX}/cmake
+#mkdir -p ${EOS_PREFIX}/scripts
+
+# install binaries 
+cp -R ${BUILD_DIR}/bin/* ${EOS_PREFIX}/bin 
+
+# install libraries
+#cp -R ${BUILD_DIR}/lib/* ${EOS_PREFIX}/lib
+
+# install cmake modules
+#sed "s/_PREFIX_/\/${SPREFIX}/g" ${BUILD_DIR}/modules/EosioTesterPackage.cmake &> ${EOS_PREFIX}/lib/cmake/${PROJECT}/EosioTester.cmake
+#sed "s/_PREFIX_/\/${SPREFIX}\/${SSUBPREFIX}/g" ${BUILD_DIR}/modules/${PROJECT}-config.cmake.package &> ${EOS_PREFIX}/lib/cmake/${PROJECT}/${PROJECT}-config.cmake
+
+# install includes
+#cp -R ${BUILD_DIR}/include/* ${EOS_PREFIX}/include
+
+# make symlinks
+#pushd ${PREFIX}/lib/cmake/${PROJECT} &> /dev/null
+#ln -sf ../../../${SUBPREFIX}/lib/cmake/${PROJECT}/${PROJECT}-config.cmake ${PROJECT}-config.cmake
+#ln -sf ../../../${SUBPREFIX}/lib/cmake/${PROJECT}/EosioTester.cmake EosioTester.cmake
```

Comment:
All these commented out lines in this file. Is it supposed to be that way?

Reply:
Yeah, when we create full binary releases with libeosio and tester hopefully all that needs to be done in the generator scripts is uncomment these lines.